packages/enzyme/src/Utils.js

```diff
@@ -308,14 +308,6 @@ export function coercePropValue(propName, propValue) {
 
   const trimmedValue = propValue.trim();
 
-  // if propValue includes quotes, it should be
```

Comment:
Why? The fact that selectors can omit quotes is imo very problematic; enforcing strings ensures that selectors used will be unambiguous.